app_flutter/lib/status_grid.dart

```diff
@@ -3,43 +3,61 @@
 // found in the LICENSE file.
 
 import 'package:flutter/material.dart';
+import 'package:provider/provider.dart';
 
 import 'package:cocoon_service/protos.dart'
     show Commit, CommitStatus, Stage, Task;
 
+import 'state/flutter_build.dart';
 import 'commit_box.dart';
 import 'task_box.dart';
 
+/// Container that manages the layout and data handling for [StatusGrid].
+///
+/// If there's no data for [StatusGrid], it shows [CircularProgressIndicator].
+class StatusGridContainer extends StatelessWidget {
+  @override
```

Comment:
This class can have a `const` constructor

Reply:
Added!